versions/1/elasticsearch/spec.yml

```diff
@@ -0,0 +1,28 @@
+spec:
+  additionalContents: false
+  contents:
+    - description: Folder containing ML model assets
```

Comment:
I understand that ML model is more related to Elasticsearch than Kibana? I'm asking about this as we used to install such resources as Kibana items like security rules, index patterns and ML module: https://github.com/elastic/package-spec/pull/152

Is this ML model different than ML module already added to Kibana?

Reply:
Hi, @mtojek - the ML model is stored in Elasticsearch, yes. It's not a Kibana item. ML modules include job configs and such that are kept track of as saved objects and then run latere so that's why they're in the Kibana asset sectioin.